src/main/java/de/dennisguse/opentracks/io/file/importer/KmzTrackImporter.java

```diff
@@ -69,20 +68,18 @@ public KmzTrackImporter(Context context, Uri uriFile) {
     public Track.Id importFile(InputStream inputStream) {
         Track.Id trackId;
 
-        if (!copyKmzImages()) {
-            cleanImport(context, importTrackId);
-            return null;
-        }
-
         try {
             trackId = findAndParseKmlFile(inputStream);
         } catch (Exception e) {
-            cleanImport(context, importTrackId);
             throw e;
         }
 
         if (trackId == null) {
```

Comment:
I guess this should not happen.
We could improve this by making findAndParseKmlFile to always throw an exception if it would not return a trackId.

Reply:
What exception should it throw if not trackId won't be returned? RuntimeException?